### ISSUE TITLE
(Re)-Design profiles

### ISSUE THREAD

**Issue by fablue** [fablue/lime-flutter#5]
## What I currently do not like
- almost everything. I hate it. 

## What I would like
- A more mature looking design. 
- It should show more information about the user 
- It should be able to be editable (in case of watching your own profile)
- It should support a nice intro animation when opening

**Comment by andprogdes** [comment]
Is it possible, that you can upload Paul?
It would be useful for screendesigns.

**Comment by fablue** [comment]
You can find logos and gerneral stuff in fablue/lime 😊
https://github.com/fablue/lime

**Comment by Giovanninono** [comment]
I wanna start a discussion with a first sketch:
I have to admit that i would also like the profile picture in the bottom left corner like its done on the community page. 

![grafik](https://cloud.githubusercontent.com/assets/29044190/26766644/18550bf0-4996-11e7-89d1-994310475637.png)

**Comment by Giovanninono** [comment]
another idea
![grafik](https://cloud.githubusercontent.com/assets/29044190/26779719/d1cb380c-49e6-11e7-9fbd-63a1ebafef51.png)

**Comment by fablue** [comment]
Pretty cool designs, but those are implying that you are able to choose some kind of header image? Would this be a cool feature instead of showing the last post in reach?

**Comment by Giovanninono** [comment]
I guess 3 different possibilities
+ editable header image
+ last post in reach 
+ most valuable post (ghosts). 

The last post in reacht would probably hit the wanted character of the app the most.
I guess a editable profile picture is enough and the flaire of the app would be obtained more and more the more impact posts in reach have.

**Comment by andprogdes** [comment]
The last idea of Giovanninono pretty good, but I would say you should delete this border around the boxes and a two-coloum layout isn't very comfortable to read on a mobile device.

Mabye the blue box, where the info about the user, can be shown with the selected color of the user.

**Comment by Giovanninono** [comment]
Yep, i was afraid that this two-column-design would not really fit especially for user, using a rather small device.

Working with your suggestioins something like that is arising.

I see a problem in clearly distinguish between the different posts on first sight but that may be because of my monitor. Or anybody got an idea how to differ the posts more cleary without a border?

![grafik](https://user-images.githubusercontent.com/29044190/26990282-635b7620-4d56-11e7-8922-157d29f8aecb.png)

**Comment by fablue** [comment]
Uuuh! I really like the idea of having tabs inside the profile (info / feed / friends)

**Comment by andprogdes** [comment]
To get a clearer seperation of the content you can use:
1. More space between the posts
2. Try if the shadow can spread into the box instead of spreading outside. If you can't imagine what I meant, search "css inset box shadow"(I know this is a web technique, but it shows really good what I think.)
http://lmgtfy.com/?q=css+box+shadow+inset
[http://lmgtfy.com/?q=css+box+shadow+inset](url)
